src/main/java/de/nadirhelix/guestbook/ResourceController.java

```diff
@@ -0,0 +1,78 @@
+package de.nadirhelix.guestbook;
+
+import java.io.IOException;
+
+import javax.annotation.Resource;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import de.nadirhelix.guestbook.resources.ResourceFacade;
+
+/**
+ * This Controller handles all requests for certain resources such as .js-files or .css-files.
+ * 
+ * @author Phil
+ */
+@RestController
+@RequestMapping("/")
+public class ResourceController {
+	
+	private static Logger log = LoggerFactory.getLogger(ResourceController.class); 
+
+	@Resource
+	private ResourceFacade resourceFacade;
+	
+	/**
+	 * Retrieves a css file based on path and fileName.
+	 * 
+	 * @param path 
+	 * 			the path where to find the file
+	 * @param fileName
+	 * 			the name of the requested file
+	 * @return {@link ResponseEntity} containing byte[] 
+	 */
+	@GetMapping("{path:.+}/css/{fileName:.+}")
+	public ResponseEntity<byte[]> getCSS(@PathVariable("path") String path, @PathVariable("fileName") String fileName) {
+		String targetPath = normalizePath(path);
+		try {
+			return ResponseEntity.ok().contentType(MediaType.TEXT_PLAIN).body(resourceFacade.getCSS(targetPath, fileName));
+		} catch (IOException e) {
+			log.debug(String.format("Could not find js file %s in path: %s", fileName, path), e);
```

Comment:
Ahh, hier ist das format. 
Das kannst du auch einfach schreiben als: log.debug("Could not find js file {} in path: {}", fileName, path, e);

Reply:
nee, die methode gibt es so leider nicht. Die Exception würde als Argument für das message format gewertet und vermutlich geschluckt werden.